openapi/components/schemas/RevenueEntry.yaml

```
@@ -0,0 +1,47 @@
+type: object
+readOnly: true
+properties:
+  customerId:
+    description: Customer's ID.
+    allOf:
+      - $ref: ./ResourceId.yaml
+    type: string
+  invoiceId:
+    description: Invoice's ID.
```

suggestion: from earlier comment

```suggestion
    description: Unique object identifier.
```

or 

```suggestion
    description: Unique invoice identifier.
```

---

```suggestion
    description: Identifier of the invoice with the item which generated the revenue.
```